src/main/java/com/minecolonies/structures/lib/RenderUtil.java

```diff
@@ -0,0 +1,65 @@
+package com.minecolonies.structures.lib;
+
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.Mirror;
+import net.minecraft.util.Rotation;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+public final class RenderUtil
+{
+
+    public static final float QUARTER = 90F;
+    public static final float HALF = 180F;
+
+    private RenderUtil()
+    {
+        throw new IllegalArgumentException("Utility Class");
+    }
+
+    public static void applyRotationToYAxis(@NotNull final Rotation rotation)
+    {
+        float angle;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "angle". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Upper-case this literal "f" suffix. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS818)